architecture/networking.adoc

```diff
@@ -9,4 +9,9 @@
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+
+Kubernetes ensures that each pod is able to network with each other, and allocates each pod an IP address from an internal network. This ensures all containers within the pod will bahave as if they were on the same host. Giving each pod its own IP address means that pods can be treated like physical hosts or virtual machines in terms of port allocation, networking, naming, service discovery, load balancing, application configuration, and migration.
+
+Creating links between pods is unneccesary. However, it is not recommended that you have a pod talk to another directly by using the IP address. Instead, it is encouraged to create a service, then interact with the service.
```

Comment:
"Instead, it is encouraged to create a service, then interact with the service."

Instead, we recommend that you create a service, then interact with the service.